pkg/controller/jobframework/reconciler.go

```diff
@@ -76,6 +76,7 @@ type Options struct {
 	// IntegrationOptions key is "$GROUP/$VERSION, Kind=$KIND".
 	IntegrationOptions map[string]any
 	EnabledFrameworks  sets.Set[string]
+	ExternalFrameworks sets.Set[string]
```

Comment:
Is this still necessary? I think you reverted the change in setup.go that needed it.

Reply:
You are correct.  Removed.  We don't currently need this.